OpenHRP_ImpedanceControllerService_impedanceParam i_param
---
bool operation_return
